src/styles/theme/components/m-virtual-displays.scss

```
@@ -86,7 +85,12 @@
       z-index: 1;
       position: relative;
       border-radius: 20px;
-      border: none;
+      transition: border-color 0.3s ease;
+      border: 2px solid #FFFFFF;
+
+      &.selected {
```

C'est plus une préférence ici, mais j'ai l'habitude d'utiliser le BEM avec SASS et donc avoir un truc du style :
```suggestion
      &--selected {
```